test/firefly/FireflyCheck.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.5.16;
+
+contract FireflyCheck {
+    uint256 dummy;
+
+    function firefly_assume(bool) external { dummy = 0; }
+    function firefly_assert(bool) external { dummy = 0; }
+
+    function firefly_newSymbolicAddress() external returns (address) { dummy = 0; }
+    function firefly_newSymbolicUint256() external returns (uint256) { dummy = 0; }
```

Comment:
Let's use `firefly_require` instead of `firefly_assume`.

Let's also use `firefly_genAddress()` and `firefly_genUInt256()` to make the same function names make sense both for random (concrete) and exhaustive (symbolic) property checking.